conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/bigmaps/BigMapsOperations.scala

```diff
@@ -339,15 +333,15 @@ case class BigMapsOperations[Profile <: ExPostgresProfile](profile: Profile) ext
     //we first extract all data available from the blocks themselves, as necessary to make a proper balance entry
     val tokenUpdates = blocks.flatMap { b =>
       val transactions = extractAppliedTransactions(b).filter {
-        case Left(op) => tokenContracts.isKnownToken(op.destination)
-        case Right(op) => tokenContracts.isKnownToken(op.destination)
+        case (Left(op), _) => tokenContracts.isKnownToken(op.destination)
```

Comment:
Here and everywhere the actual operation group hash is not used eventually, I'd first thing get the list and map to only keep the useful content, discarding the hash.
Doing that should simplify any computation downstream.